components/workspaces/production/stone-prd-host1/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../base/
+- route.yaml
+images:
+- name: quay.io/konflux-workspaces/workspaces-server
+  newTag: v0.1.0-alpha5
+- name: quay.io/konflux-workspaces/workspaces-operator
+  newTag: v0.1.0-alpha5
+
+configMapGenerator:
+- behavior: merge
+  literals:
+  - log.level=-4
```

Comment:
-4 may be too low, I'd suggest `0` (https://pkg.go.dev/log/slog#Level)

Reply:
Fixed in 886ad32.